### ISSUE TITLE
Sanity Checks scripts

### ISSUE THREAD

**Issue by bourdeet** [alexandrainst/alexandra-iotml-template#17]
It would be helpful to include, in addition to `train_model` and `eval_model`, some standard scripts for assuring that the data is properly processed, and that the training went well. These scripts should include at the very least:

pre-training:
- histogram distributions of a dataset being piped into a model

post-training - maybe as part of `eval_model`:
- Correlation coefficients of the latent space variables (in the case of AE families of model)
- loss function over time (already implemented, though maybe not at the right place in the code
- Latent space encoding wrt. multivariate gaussian (if using a VAE)
- reconstruction loss wrt. training / test data

Evaluation against actual anomalies
- outlier latent space detection
- timecoded evolution of anomalies in latent space
- alert response time & false alarm rate vs. selected alarm threshold conditions
